Applications/MNIST/README.md

```diff
@@ -41,18 +41,25 @@ int getBatch_train(float **inVec, float **inLabel, bool *last,
 The model configuration is in ```res/mnist.ini```. ADAM optimizer is used and the trained model is going to be saved in ```./model.bin```.
 
 ``` bash
-# Model Section : Model
+# Network Section : Network
+
 [Model]
-Type = NeuralNetwork	    # Network Type NeuralNetwork
-Learning_rate = 1e-4 	    # Learning Rate
-Epochs = 1500		        # Epochs
-Optimizer = adam 	        # Optimizer : adam (Adamtive Moment Estimation)
-Loss = cross  		        # Loss function : cross ( for cross entropy )
-Save_Path = "model.bin"  	# model path to save / read
-batch_size = 32		        # batch size
-beta1 = 0.9 		        # beta 1 for adam
-beta2 = 0.999	            # beta 2 for adam
-epsilon = 1e-7	            # epsilon for adam
+Type = NeuralNetwork	            # Network Type : Regression, KNN, NeuralNetwork
+Epochs = 1500		                # Epochs
```

Comment:
How about matching the indentation with other '#'s?
and LGTM

Reply:
That's not what I meant. what I saw was different because of the editor. 
Thank you.!!!